sqlobject/src/main/java/org/jdbi/v3/sqlobject/ResultReturnThing.java

```diff
@@ -55,9 +56,13 @@ static ResultReturnThing forType(ResolvedMethod method)
                     "Method %s#%s is annotated as if it should return a value, but the method is void.",
                     method.getDeclaringType().getErasedType().getName(),
                     method.getName()));
-        } else if (return_type.isInstanceOf(ResultBearing.class)) {
+        }
+        else if (return_type.isInstanceOf(ResultBearing.class)) {
             return new ResultBearingResultReturnThing(method);
         }
+        else if (return_type.isInstanceOf(Stream.class)) {
```

Comment:
Still want to clean up this ResultReturnThing pile, but we'll do that separately later.